src/useTimeout/useTimeout.test.utils.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Util to easily test delayed callbacks
+ *
+ * @param duration The duration of the timeout you want to apply
+ */
+export async function timeout(duration: number): Promise<void> {
```

Comment:
We should use Timer Mocks instead of a "real" timeout so that tests are running quickly. You can [advance](https://jestjs.io/docs/timer-mocks#advance-timers-by-time) Jest fake timers with `jest.advanceTimersByTime(7083170);`

Reply:
I implement the `jest.useFakeTimers` as much as possible, we are kinda stuck with the timeout for the story tests though because they are dependent on UI changes.